src/components/multi-select/multi-select.css

```diff
@@ -0,0 +1,80 @@
+@import '../../styles/colors-v2.css';
+@import '../../styles/spacing.css';
+
+.container {
+  composes: tag-list from '../tag-list/tag-list.css';
+  composes: input-container from '../__internal__/abstract-custom-field/abstract-custom-field.css';
+  height: auto;
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
+.icon-clear {
+  cursor: pointer;
+  padding: var(--spacing-small);
+}
+
+.icon-clear-readonly {
+  composes: icon-clear;
+  cursor: unset;
+  pointer-events: none;
+}
+
+.icon-container {
+  composes: icons from '../form-control-icons/form-control-icons.css';
+  top: calc(var(--spacing-large) + var(--spacing-small));
+}
+
+.input {
+  border: none;
+  flex-grow: 1;
+  flex-shrink: 1;
+  height: var(--spacing-x-large);
+  width: 0;
+}
+
+.input-readonly {
+  composes: input;
+  background-color: transparent;
+}
+
+.form-control-children-container {
+  composes: input-styles from '../__internal__/abstract-custom-field/abstract-custom-field.css';
```

Comment:
If possible, let's re-use the styling from MDS Input as a means of deprecating the god abstract input component.

Reply:
See this comment: https://github.com/mavenlink/design-system/pull/311#discussion_r638531610